lib/influxer/metrics/metrics.rb

```diff
@@ -45,6 +46,12 @@ def attributes(*attrs)
         end
       end
 
+      def values(*attrs)
```

Comment:
We've already have attributes, which are the same as values. So we don't need this new feature.